postgres/src/main/java/org/jdbi/v3/postgres/PgLobImpl.java

```diff
@@ -0,0 +1,66 @@
+package org.jdbi.v3.postgres;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.postgresql.PGConnection;
+import org.postgresql.largeobject.LargeObject;
+import org.postgresql.largeobject.LargeObjectManager;
+
+class PgLobImpl implements PgLob {
+    private static final int BUF_SIZE = 1024 * 64;
```

Comment:
Any particular reason for this exact number? Wouldn't this make a good Config value?

Reply:
No, it's just picked as a "big enough" value.  I aligned it to the Postgres driver which uses 4K.
It's hard coded in the driver so we don't really have the ability to tune it properly, so we shouldn't expose configuring as API at this time.